yarn.lock

```diff
@@ -2682,41 +2682,20 @@
   resolved "https://registry.yarnpkg.com/@sindresorhus/is/-/is-0.14.0.tgz#9fb3a3cf3132328151f353de4632e01e52102bea"
   integrity sha512-9NET910DNaIPngYnLLPeg+Ogzqsi9uM4mSboU5y6p8S5DzMTVEsJZrawi+BoDNUVBa2DhJqQYUFvMDfgU062LQ==
 
-"@sinonjs/commons@^1.6.0", "@sinonjs/commons@^1.7.0", "@sinonjs/commons@^1.8.3":
+"@sinonjs/commons@^1.7.0":
```

Comment:
Can you checkout `origin/develop` on this one and rerun the `yarn add` / `yarn install` commands? I tried it locally and I don't get new `yarn.lock` entries

Reply:
sure

Reply:
origin/develop or origin/main?

Reply:
`origin/main` since you have it as destination. You're faster than the snipe edit :D

Reply:
You're right that shouldn't be changed, commits on their way!